### PR TITLE
Update README.md and CONTRIBUTING.md

### LINKED ISSUE #236: [BUG] Helm instructions not available
**Describe the bug**
For any new contributors, the [CONTRIBUTING.md](https://github.com/infracloudio/botkube/blob/develop/CONTRIBUTING.md) file goes through a detailed step-by-step to get the project set up on everyone's local machine. One can decide to run the project locally, or through docker. In the latter, however, a reference is made to the [README.md](https://github.com/infracloudio/botkube#using-helm) on how to use `helm`. 

This, however, seems to be removed. The link is also referencing to a header `#using-helm`, which does not exist in the README. Now the docker setup can't easily be completed by anyone joining the project.

**To Reproduce**
1. Go to [CONTRIBUTING.md](https://github.com/infracloudio/botkube/blob/develop/CONTRIBUTING.md#build-the-container-image)
2. pass through to [the link provided](https://github.com/infracloudio/botkube#using-helm).
3. Notice the lack of information given

**Expected behavior**
Either add the setup and complete usage of `helm` directly on the CONTRIBUTING.md, or reinstate it in the README.md.

### PR DESCRIPTION
**Pull request by PrasadG193**
##### ISSUE TYPE

 - Docs Pull Request

##### SUMMARY

- Add steps to install BotKube from develop branch
- Update contributing doc to work with the latest changes 

Fixes #236

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,8 +8,50 @@ You can also ask BotKube to execute kubectl commands on k8s cluster which helps
...
+--set resourceConfig.settings.clustername=<CLUSTER_NAME> \
+--set resourceConfig.settings.allowkubectl=<ALLOW_KUBECTL> \
+--set image.repository=infracloudio/botkube \
+--set image.tag=latest \
+infracloudio/botkube
+```
+
+Check [values.yaml](https://github.com/infracloudio/botkube/blob/develop/helm/botkube/values.yaml) for default options
+
+> Note: <br>
```
Comment (bhavin192) [inline README.md]:
```suggestion
> Note:  
```
Adding two spaces will add a newline

File: README.md [hunk 2]
```diff
@@ -9,8 +8,50 @@ You can also ask BotKube to execute kubectl commands on k8s cluster which helps
...
+$ helm repo update
+$ kubectl create namespace botkube
+$ helm install --version v9.99.9-dev botkube --namespace botkube \
+--set communicationConfig.communications.slack.enabled=true \
+--set communicationConfig.communications.slack.channel=<SLACK_CHANNEL_NAME> \
+--set communicationConfig.communications.slack.token=<SLACK_API_TOKEN_FOR_THE_BOT> \
+--set resourceConfig.settings.clustername=<CLUSTER_NAME> \
+--set resourceConfig.settings.allowkubectl=<ALLOW_KUBECTL> \
+--set image.repository=infracloudio/botkube \
+--set image.tag=latest \
```
Comment (bhavin192) [inline README.md]: I think it will look cleaner if we have some indentation here

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -93,26 +75,25 @@ Now you can build and run BotKube by one of the following ways
 
 ## Making A Change
 
-* *Before making any significant changes, please [open an
-issue](https://github.com/infracloudio/botkube/issues).* Discussing
-your proposed changes ahead of time will make the contribution process
-smooth for everyone.
+* Before making any significant changes, please [open an issue](https://github.com/infracloudio/botkube/issues).
+
+* Discussing your proposed changes ahead of time will make the contribution process smooth for everyone.
```
Comment (bhavin192) [inline CONTRIBUTING.md]: I think these two should be one point. What do you think?

Comment (PrasadG193) [reply CONTRIBUTING.md]: Sounds good

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -2,36 +2,33 @@
...
+Before you proceed, make sure you have installed BotKube Slack/Mattermost app and copied required token as per the steps documented [here](https://www.botkube.io/installation/)
+
 ### Prerequisite
-* Make sure you have `go` compiler installed.
-* BotKube uses [`dep`](https://github.com/golang/dep) to manage
-dependencies.
-* You will also need `make` and
-[`docker`](https://docs.docker.com/install/) installed on your
+
+* Make sure you have `go 1.11+` installed with go module activated. (You can set env var `GO111MODULE=on/auto` to activate)
```
Comment (sanketsudake) [inline CONTRIBUTING.md]: Separate this out as `GO111MODULE=on` or `GO111MODULE=auto`

File: README.md [hunk 5]
```diff
@@ -9,8 +8,52 @@ You can also ask BotKube to execute kubectl commands on k8s cluster which helps
...
 
-## Getting started
-Please follow [this](https://www.botkube.io/installation/) for complete BotKube installation guide.
+## Installation
+
+### Stable release (Recommended)
+
+Please follow [this guide](https://www.botkube.io/installation/) to install latest stable version of BotKube.
+
+### From develop branch
```
Comment (sanketsudake) [inline README.md]: We should separate out this section may be to contributors doc. User might end up copying following steps when they actually want to use stable branch.

**Pull request merged**